Source/NSPropertyList.m

```diff
@@ -345,7 +345,7 @@ - (void) parser: (NSXMLParser *)parser
     }
   else if ([elementName isEqualToString: @"real"])
     {
-      ASSIGN(plist, [NSNumber numberWithDouble: [value doubleValue]]);
+      ASSIGN(plist, [NSNumber numberWithDouble: strtod([value cString], 0)]);
```

Comment:
Cool! A minor (teensy, in fact) nitpick, though: The type of the second parameter to `strtod` is also a pointer, so `NULL` would be more appropriate than `0`.

Reply:
```suggestion
      ASSIGN(plist, [NSNumber numberWithDouble: strtod([value cString], NULL)]);
```